lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1793,6 +1793,29 @@ definitions:
                 example:
                   1: ['192.168.0.0/16']
                   2: ['172.10.0.0/16', '172.20.0.0/16']
+      restricted_plmn_ids:
+        type: array
+        description: 'List of PLMN IDs restricted in the network'
+        items:
+          type: string
+          minLength: 5
+          maxLength: 6
+          pattern: '^(\d{5,6})$'
+          example: '123456'
```

Comment:
can we make this a list of plmn id objects and a plmn id object has mcc and mnc parts.